airqo-platform/AirQo-api
# updated the partial URL for candidate registration

- What does this PR do?
This PR just updates the backend endpoint for user registration
- What JIRA task is related to this PR?
No JIRA card
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/auth-service` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run dev` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
**base url:** http://localhost:3000/api/v1/users
[**POST**] [**registerCandidate**] **partial url:** _/register/new/candidate_

More details: [API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=2081265667) in the **auth-service** sheet
